stanford/suclass/ansible.cfg

```diff
@@ -1,7 +1,7 @@
 [defaults]
 callback_whitelist = profile_tasks
 host_key_checking = False
-inventory = ../../playbooks/edx-east/ec2.py
+inventory = inventory.yaml
```

Comment:
this seems reasonable

Reply:
It's not just "reasonable"; it's what we're doing everywhere :)